python/test/fuzzing_test.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Python GLOME fuzz main.
+
+This module that implement some easy fuzz testing for pyglome. 
+"""
+
+from hypothesis import given, settings, strategies, example
```

Comment:
pystyle rule 2.2 "Use import statements for packages and modules only, not for individual classes or functions."

Reply:
Fixed